<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BurnCpu</class>
 <widget class="QWidget" name="BurnCpu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="centralWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>119</x>
        <y>22</y>
        <width>343</width>
        <height>457</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="cpuusagetext">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777213</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>22</pointsize>
              </font>
             </property>
             <property name="text">
              <string>CPU:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="cputemperaturetext">
             <property name="font">
              <font>
               <pointsize>22</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Temp.</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="time">
             <property name="font">
              <font>
               <pointsize>22</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Time:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="cpuusagevalue">
             <property name="font">
              <font>
               <pointsize>22</pointsize>
              </font>
             </property>
             <property name="text">
              <string>%</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="cputempvalue">
             <property name="font">
              <font>
               <pointsize>22</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Â°C</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLineEdit" name="lineEdit_timerH">
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>30</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>22</pointsize>
                </font>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_9">
               <property name="font">
                <font>
                 <pointsize>22</pointsize>
                </font>
               </property>
               <property name="text">
                <string>:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_timerM">
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>30</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>22</pointsize>
                </font>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_10">
               <property name="font">
                <font>
                 <pointsize>22</pointsize>
                </font>
               </property>
               <property name="text">
                <string>:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_timerS">
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>30</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>22</pointsize>
                </font>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="font">
            <font>
             <pointsize>22</pointsize>
            </font>
           </property>
           <property name="text">
            <string>START</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_removefile">
           <property name="font">
            <font>
             <pointsize>22</pointsize>
            </font>
           </property>
           <property name="text">
            <string>REMOVE FILE</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="pushButton_startTimer">
           <property name="font">
            <font>
             <pointsize>22</pointsize>
            </font>
           </property>
           <property name="text">
            <string>start Timer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_stopTimer">
           <property name="font">
            <font>
             <pointsize>22</pointsize>
            </font>
           </property>
           <property name="text">
            <string>stop Timer</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>File Content</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="textBrowser"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QToolBar" name="mainToolBar"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
